dlrover/python/master/node/local_job_manager.py

```diff
@@ -60,9 +60,11 @@ def early_stop(self):
     def add_node_event_callback(self, node_event_callback):
         pass
 
-    def update_node_resource_usage(self, node_type, node_id, cpu, memory):
+    def update_node_resource_usage(
```

Comment:
Add a docstring to explain what is the gpu_stats. You can give an example of gpu_stats in the docstring.

Reply:
Update the docstring in `NodeResource` like this：
```
class NodeResource(JsonSerializable):
    """NodeResource records a resource of a Node.
    Attributes:
        cpu: float, CPU cores.
        memory: float, memory MB.
        gpu_type: str, the type of GPU.
        gpu_num: int,
        gpu_stats: list of GPUMetric, a list of GPUMetric dataclass objects,
        each containing GPU statistics.
            - index (int): The index of the GPU.
            - total_memory_mb (int): Total GPU memory in megabytes.
            - used_memory_mb (int): Used GPU memory in megabytes.
            - gpu_utilization (float): GPU utilization in percentage (0.0 to
              100.0).
        image: the image name of the node.
        priority: the priority classs of the node.
    Example:
        To update the resource usage of a worker node with ID 123 and provide
        GPU stats:
        >>> gpu_stats = [GPUMetric(index=0, total_memory_mb=8192,
        used_memory_mb=2048, gpu_utilization=80.0)]
        >>> cluster_manager.update_node_resource_usage('worker', 123, 75.0,
        50.0, gpu_stats)
    """
```
or
Add new docstring to `update_node_resource_usage`?

Reply:
Updation of `NodeResource`.